### PR TITLE
www.naughtymachinima.com

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/102685

### CODE REVIEW

File: EnglishFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -1815,6 +1815,7 @@ news18.com#?##right > div[style="height:250px;"]:has(> div.expandoad)
 news18.com#?#.recomeded_story > ul > li:has(> div[style]> div[class*="dfp-ad"])
 addmusictophoto.com#?#h6:contains(/^Advertisement$/)
 fromtexttospeech.com#?#h2:contains(/^Advertisement$/)
+naughtymachinima.com#?#body > div[style]:has(> canvas)
```
Comment (zloyden) [inline EnglishFilter/sections/css_extended.txt:1818]: Can you show this with opened devtools? I don't see this on Chrome / Safari.

Comment (jellizaveta) [reply EnglishFilter/sections/css_extended.txt:1818]: <details><summary>Screenshot:</summary>

Снимок экрана 2021-12-28 в 15 28 03

</details><br/>

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/css_extended.txt:1818]: @jellizaveta 
I can't reproduce this element neither but try the rule under this comment

! dancers `Popping Tool` (domain.com/t28193693144.js)

Comment (jellizaveta) [reply EnglishFilter/sections/css_extended.txt:1818]: @Sergey-Lyapin not work on my end

**Pull request closed without merging**